### ISSUE TITLE
Does isochron work on LS1028 switch port?

### ISSUE THREAD

**Issue by xinliwang1992** [vladimiroltean/isochron#22]
Hi, @vladimiroltean 
The NXP ls1028 board has two TSN ports, one is ENETC and others are DSA.
The isochron works on the ENECT port but it seems not working on the DSA ports. So, I wonder if the isochron works on LS1028 switch port?
I found out that the TIMESTAMPING can not achived by setting setsockopt(fd, SOL_SOCKET, SO_TIMESTAMPING, &flags, sizeof(flags)).
Thanks

**Comment by vladimiroltean** [comment]
No, you cannot make the switch ports on LS1028A timestamp non-PTP packets, which isochron requires.
